metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/monitoring/HiveMetrics.java

```diff
@@ -67,19 +69,20 @@
     listPartitionsByFilter("listPartitionsByFilter"),
     getPartitions("getPartitions"),
     getPartitionCount("getPartitionCount"),
+    getPartitionKeys("getPartitionKeys"),
```

Comment:
We need to make sure we follow a naming convention for the metrics. Will the metric start with metacat.?